downloader.py

```diff
@@ -62,13 +62,19 @@ def download_file(filename, prefix, num_videos=None):
         print(e)
 
     num_downloaded = 0
+    available_files = os.listdir('{}_videos'.format(prefix))
 
     with open(filename) as csv_file:
+        # Format of csv files is: YouTube ID, start segment, end segment, X coordinate, Y coordinate
         csv_reader = csv.reader(csv_file, delimiter=',')
-        for row in csv_reader:
+        for i, row in enumerate(csv_reader):
```

Comment:
Is i used for anything?

Reply:
I used i to start downloading from a certain row in the CSV file. For example, if I previously downloaded 800 entries and wanted to continue from there.

Reply:
Is the code to do that still in here? If it's not, I think we should take it out.